apps/teamsjs-cc-app/tabs/src/helpers/index.ts

```diff
@@ -0,0 +1,9 @@
+export const booleanToString = (value: boolean) => {
+  return value ? "Yes" : "No";
+};
+
+export const convertRestIdToEwsId = (restId: String) => {
```

Comment:
A small comment here explaining what this is doing and why it's needed would be helpful